### PR TITLE
Update choosy from 1.3 to 2.0

### PR DESCRIPTION
**Pull request by eenick**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

**Comment by eenick** [comment]
I'm going to need some help here fixing this cask.

The previous 1.3 version was a system preference pane and is designed for OS X 13.0 (High Sierra) and below. 

The newer 2.0 version is now a standalone app and is designed for OS X 14.0 (Mojave) and up. 

See [](https://www.choosyosx.com/releases/2.0) for details.

File: Casks/choosy.rb [hunk 1]
```diff
@@ -3,8 +3,8 @@
     version '1.1'
     sha256 'c6530d4e0dddbf47c6a8999bda8f3a5ef1857f4481b9325e56cfe00f05b2022c'
   else
```
Comment (ran-dall) [inline Casks/choosy.rb]: Replace with this, starting from line 5...

```
  else if MacOS.version <= :high_sierra
    version '1.3'
    sha256 'cb1f40df11ac1b52354f4b81367462d2646a6d023c64bafe5022fcec52f796cd'
  else
    version '2.0'
    sha256 'dcba0b48d742da6c6860858c81480ff8b08b7cdbaae10a348c3062afddc1cc73'
```

**Review by ran-dall — CHANGES_REQUESTED** [review]
Replace with this, starting from line 5...

```
  else if MacOS.version <= :high_sierra
    version '1.3'
    sha256 'cb1f40df11ac1b52354f4b81367462d2646a6d023c64bafe5022fcec52f796cd'
  else
    version '2.0'
    sha256 'dcba0b48d742da6c6860858c81480ff8b08b7cdbaae10a348c3062afddc1cc73'
```

**Comment by ran-dall** [comment]
@reitermarkus 🙏

**Pull request merged**